src/notation/view/widgets/editstyle.cpp

```diff
@@ -1473,6 +1474,60 @@ QString EditStyle::pageCodeForElement(const EngravingItem* element)
     }
 }
 
+QString EditStyle::subPageCodeForElement(const EngravingItem* element)
+{
+    IF_ASSERT_FAILED(element) {
+        return QString();
+    }
+
+    switch (element->type()) {
+    case ElementType::TEXT: {
```

Comment:
I believe another set of indents is required between `switch` and `case` (same applies in other places)